cmd/ccboc/cmd/commands.go

```diff
@@ -364,6 +366,90 @@ func request(method, endpoint string, buffer *bytes.Buffer) ([]byte, *errorRespo
 	return body, nil, nil
 }
 
+func getCalculationPhase() error {
+	for {
+		app := tview.NewApplication()
+		table := tview.NewTable().
+			SetBorders(true)
+
+		calculationBulk, err := getCalculationBulkData()
+		if err != nil {
+			logrus.WithError(err).Fatal()
+		}
+
+		populateTable(calculationBulk, table)
+
+		table.Select(0, 0).SetFixed(1, 1).SetDoneFunc(func(key tcell.Key) {
+			if key == tcell.KeyEnter { // to refresh the table contents
+				app.Stop()
+			}
+			if key == tcell.KeyEsc { // to exit the app
+				app.Stop()
+				os.Exit(0)
```

Comment:
Do we need this?

Reply:
This needs to be done to correctly exit from the app.

Reply:
What will happen after `app.Stop()` ?

Reply:
If you exit here, it seems that we will never clear the table, which is the code below, but now it is unreachable.